src/__tests__/__snapshots__/settingsOverview.test.js.snap

```diff
@@ -803,14 +795,6 @@ exports[`MyComponent should render UnLocked SettingsOverview 1`] = `
         disabled={false}
         leftText="Ufo"
       />
-      <RowRoute
-        disabled={false}
-        leftText="Smartcash"
-      />
-      <RowRoute
-        disabled={false}
-        leftText="Groestlcoin"
-      />
```

Comment:
Why are we removing SmartCash and Groestlcoin?

Reply:
So I am not sure why the test removed this, but this is in the test only it was auto generated snap shot